File: src/main/java/io/bloco/faker/components/University.java
package io.bloco.faker.components;

import io.bloco.faker.FakerComponent;
import io.bloco.faker.FakerData;

public class University extends FakerComponent {

    public University(FakerData data) {
        super(data);
    }

    public String name() {
        return parse(sample("name"));
    }

    public String prefix() {
        return sample("prefix");
    }

    public String suffix() {
        return sample("suffix");
    }
}

File: lib/src/main/kotlin/io/bloco/faker/components/University.kt
package io.bloco.faker.components

import io.bloco.faker.FakerComponent
import io.bloco.faker.FakerData

class University(data: FakerData) : FakerComponent(data) {
    fun name(): String {
        return parse(fetch("university.name"))
    }

    fun prefix(): String {
        return fetch("university.prefix")
    }

    fun suffix(): String {
        return fetch("university.suffix")
    }
}
